samples/bluetooth/mesh/light_dimmer/README.rst

```diff
@@ -94,6 +98,11 @@ Button 1:
 
    On press and hold, **Button 1** will publish a Generic Level move set message using the configured publication parameters of its model instance and will continuously dim the LED lightness state on a :ref:`mesh light fixture <bluetooth_mesh_light_lc>` device. On release, the button publishes another Generic Level move set message with the ``delta`` parameter set to 0 and stops the continuous level change from the previous message.
 
+Button 1:
+   On short press, **Button 2** will publish a Scene Restore message using the configured publication parameters of its model instance, and restore the LED state of all the targets to the values stored under the current scene number. Each press of the button will recall the next scene, meaning, the first press will recall scene 1, the next press will recall scene 2, and so on, before wrapping around back to scene 0.
```

Comment:
Wrap around back to scene 1 maybe? No mention of a scene 0 before

Reply:
This is indeed wrong, but not for the reason you say. It _does_ wrap to scene 1, but the first press recalls scene 2. This is the way it works because it has to be "on" scene 1 at boot for the store operation on fresh boot to make sense (store to scene 1).